pkgs/applications/graphics/xpano/default.nix

```diff
@@ -0,0 +1,63 @@
+{ lib
+, stdenv
+, fetchurl
+, cmake
+, pkg-config
+, ninja
+, opencv
+, SDL2
+, gtk3
+, catch2_3
+, spdlog
+, exiv2
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xpano";
+  version = "0.16.1";
+
+  src = fetchurl {
+    url = "https://github.com/krupkat/${pname}/releases/download/v${version}/${pname}-v${version}.tar.gz";
```

Comment:
Can we use fetchFromGitHub instead?

Reply:
Yes, it can work with fetch submodules = true, the archive on the url has the submodules packaged in. Is the fetchFromGithub preferred? I wasn't sure from the docs.

And thanks for the review comments! I'll update the pr in the morning.

Reply:
I have switched to fetchFromGitHub.